.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "trailingComma": "none",
+    "singleQuote": true,
+    "semi": false
```

Comment:
This is a bad idea, make semi mandatory. This leads to unintended bugs in production.
Simple example:
```js
console.log(123)
(function {
  // some logic
})()
```
Provided code won't work as minified version looks like so:
```js
console.log(123)(function() {/*...*/}())
```
Now the IIFE is being curried by the log from the console which results an error so all the following code won't be executed. Good luck on spotting that issue on production.